pygmt/datasets/load_earth_dataset.py

```diff
@@ -0,0 +1,177 @@
+"""
+Internal function to download remote Earth datasets.
+"""
+from typing import Dict, NamedTuple
+
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import kwargs_to_strings
+from pygmt.io import load_dataarray
+from pygmt.src import grdcut, which
+
+
+class Resolution(NamedTuple):
+    """
+    The available grid registrations for a given resolution and whether it is a
+    tiled grid.
+    """
+
+    registrations: list[str]
+    tiled: bool
+
+
+class Dataset(NamedTuple):
+    """
+    Standard information about a dataset and grid metadata.
+    """
+
+    dataset_title: str
```

Comment:
Remove the prefix `dataset_`, because other attributes also don't have it.
```suggestion
    title: str
```